src/changelog.js

```diff
@@ -64,14 +56,35 @@ async function getChangelog({ repository }) {
     'history.md',
     'HISTORY',
     'history',
-  ].map(file => [baseUrl.replace(/\/$/, ''), file].join('/'));
+  ];
 
-  try {
-    const { url } = await race(files.map(got, { method: 'HEAD' }));
-    return { changelogFilename: url };
-  } catch (e) {
-    return { changelogFilename: null };
+  const unpkgFiles = fileOptions.map(
+    file => `${config.unpkgRoot}/${name}@${version}/${file}`
+  );
+
+  if (repository === null) {
+    return raceFromPaths(unpkgFiles);
+  }
+
+  const user = repository.user || '';
+  const project = repository.project || '';
+  const host = repository.host || '';
+  if (user.length < 1 || project.length < 1) {
+    return raceFromPaths(unpkgFiles);
+  }
+
+  // Check if we know how to handle this host
+  if (!baseUrlMap.has(host)) {
```

Comment:
we can put unpkg in the base url map & keep it as-is maybe?

Reply:
I thought about putting it in the baseUrlMap, but it doesn't quite work in there - it would require passing `name` and `version` into the `baseUrlMap` which doesn't make sense for anything else.

What's the thing you want to change about the code as-is?

Reply:
makes sense, let's keep it like in your OR